e2e-tests/playwright/specs/functional/system_console/permissions/team_access.spec.ts

```diff
@@ -0,0 +1,176 @@
+    await adminConsolePage.goto();
+
+    // Login to the System Console and navigate to Delegated Granular Administration
+    await adminConsolePage.sidebar.goToItem('Delegated Granular Administration');
+
+    // Find the System Manager row in the table
+    const systemManagerText = adminConsolePage.page.getByText('System Manager', {exact: true}).first();
+    await expect(systemManagerText).toBeVisible();
+
+    // Click on the System Manager text to go to its settings page
+    await systemManagerText.click();
+
+    // Expand the section
+    const sectionReporting = adminConsolePage.page.locator(`data-testid=${sectionTestId}`);
+    const hideSubsectionsLink = sectionReporting.getByRole('button').filter({hasText: 'Hide'}).first();
+    const showSubsectionsLink = sectionReporting.getByRole('button').filter({hasText: 'Show'}).first();
+
+    // Check which one is visible and click if needed
+    const isHideVisible = await hideSubsectionsLink.isVisible();
+    const isShowVisible = !isHideVisible && (await showSubsectionsLink.isVisible());
+
+    if (isShowVisible) {
+        // Need to expand
+        await showSubsectionsLink.click();
+    }
+
+    // Get the whole row
+    const rowReporting = adminConsolePage.page.locator('.PermissionRow').filter({has: sectionReporting});
+    await rowReporting.click();
+
+    // Find the sub section
+    const subsectionTeamStatistics = rowReporting.locator(`data-testid=${subsectionTestId}`);
+
+    await subsectionTeamStatistics.click();
+
+    // Look for dropdown button
+    const dropdownButton = subsectionTeamStatistics.locator('button').first();
+
+    // Click the button to open the dropdown menu
+    await dropdownButton.click();
+
+    // Click on the desired option in the dropdown
+    const permissionOption = subsectionTeamStatistics.locator('.dropdown-menu').getByText(permissionText).first();
+    await permissionOption.click();
+
+    // Click Save button
+    const saveButton = adminConsolePage.page.getByRole('button', {name: 'Save'}).first();
+    await saveButton.click();
+
+    // Wait for save operation to complete
+    await adminConsolePage.page.waitForLoadState('networkidle');
+
+    // Go back to the main console
+    await adminConsolePage.goto();
+    await adminConsolePage.page.waitForLoadState('networkidle');
+};
+
+test('MM-63378 System Manager without team access permissions cannot view team details', async ({pw}) => {
+    const {adminUser, adminClient, user: systemManagerUser, userClient: systemManagerClient} = await pw.initSetup();
+
+    // Create a user with the system_manager role
+    // const systemManagerUser = await adminClient.createUser(pw.random.user(), '', '');
+    await adminClient.updateUserRoles(systemManagerUser.id, 'system_user system_manager');
+
+    // Create a Team and add the user as a member
+    const team = await adminClient.createTeam(pw.random.team());
+    await adminClient.addToTeam(team.id, systemManagerUser.id);
```

Comment:
You may use the `team` from `initSetup` where such user is already added, like

```suggestion
    const {
        adminUser,
        adminClient,
        user: systemManagerUser,
        userClient: systemManagerClient,
        team,
    } = await pw.initSetup();

    // Update user with system_manager role
    await adminClient.updateUserRoles(systemManagerUser.id, 'system_user system_manager');
```

Reply:
Nice, thank you!!!